### DISCUSSION TITLE
Docker Image + Studio issues - api/projects/default/api/rest 404 & api/profiles/permissions

### DISCUSSION THREAD

**Discussion by DEPSTRCZ** [supabase/supabase#12768] [Questions]
My setup:
- Docker compose
- Nginx Reverse proxy

I can acces the page.. Browse but it wont connect to my project (Default)

There has been an issue with the ``api/profiles/permissions`` retruning 404.. Which has been hopefully fixed in the newest studio image..
So i changed the docker compose file to the newest fixed image.. The`` api/profiles/permissions`` issue was fixed, but a newerror appeared..
``HEAD https://reverseproxy.domain.com/api/projects/default/api/rest 404 (Not Found)``
And i`m unsure what to do.

**Comment by ruggi99** [comment]
You're right, it doesn't work because of that endpoint doesn't exists in self-hosted.
Supabase doesn't take care of self-hosted, they take care only of their platform.
Not the first time they break things like this in self-hosted.
You have to send a PR or wait for someone to fix this.
You can try sending a issue but they quite never answer ultimately.
It's frustrating I know.

EDIT:
I tried sending some PRs to add tests to try to fix this behaviour but they have declined them

**Reply by DEPSTRCZ** [reply]
Thats sad... I wanted to use supabase but it wont load :/
Looks like i will have to find something else.

**Reply by ruggi99** [reply]
Not really. You can stick to a previous version for a bit until they fix. You should watch PRs and see if someone tries to fix

**Reply by iamkhalidbashir** [reply]
@ruggi99 Do you know why my studio (docker) doesn't show edge functions logging on the side bar? I just ran docker-compose up in the docker directory, but it doesn't have edge functions in it the studio dhasboard.
